docs/blog/2019-03-05-dot-org-messaging-survey/index.md

```diff
@@ -0,0 +1,131 @@
+- Low to no hosting costs: Gatsby eliminates the need for traditional hosting costs because the site is hosted on the edge (Content Delivery Network).
+  Instantly scalable: No servers and no databases to manage with a website that never goes down because it’s built on the edge, instantly scaling for traffic spikes.
+- Secure: Gatsby websites are inherently more secure because there are no servers involved. All your content is compiled ahead of time so hackers can’t get to your database or into your CMS. And your site is more resilient to DDoS attacks because the biggest point of failure — a server — is no longer part of the equation
+
+## Extra values
+
+Here are some more values that community members listed that we hadn’t listed in the survey: a welcoming community, plugins, accessibility matter to Gatsby users, and that Gatsby is fun AND easy to use. It’s good to know these things are valuable because then we can keep building on our successes and refining Gatsby’s ability to provide this value.
+
+Community
+
+> The community around Gatsby is outstanding! From the store, all the public events, Twitter, podcast appearances, insane activity in the repository.. anytime I've needed help I've been able to easily find it, or get it from one of the above.
+> Plugins
+> Plugin ecosystem: it's growing quickly and it's a nice way to contribute and engage with others. I've met some great folks because of it
+> Plugins and ability to use npm gives you a huge ecosystem to draw on.
+> Accessibility
+> Gatsby allows for accessible websites by statically rendering pages, and also makes use of Reach Router to ensure a more accessible experience. A lot of modern JavaScript applications result in poor accessibility, but Gatsby takes care of important core functionality that doesn't force compromise.
+> Easy & fun
+> “Fun”! I really enjoy working with Gatsby. It work they way it should and sometimes even exceeds my expectations.
+
+## Takeaways
+
+The most concrete next step is to redesign the .org homepage with what we learned from this survey.
+
+Here are some ideas to keep in mind throughout for a potential redesign of the .org site:
+
+- If we stick with “blazing”, it will be a bit controversial and it might help to change the tone to playful or lean into it somehow
+- If we say Gatsby is a framework, we will need to define that word since Gatsby’s history is as a static site generator
+- We should give examples of how fast Gatsby sites are, because speed is relative.
+- We ought to emphasize and build on these three values: fast sites, performance out of the box, productive from the start. If we don’t go with that, we need to be aware that we are teaching people something new that they don’t necessarily expect, which is fine if we want to do it!
+
+Quote:
+
+> “[I didn’t think the value Gatsby provides was] that crucial until I experienced it first hand.”
```

Comment:
You can use the new `<pullquote>` component eg: https://github.com/gatsbyjs/gatsby/pull/11416/files#diff-b0fa294bd703315607000dca353de775R38